.github/workflows/docker-publish.yml

```diff
@@ -22,6 +22,17 @@ jobs:
           # ensure latest base image is used
           pull: true
           push: false
+          tags: adaguc-server
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.24.0
+        with:
+          image-ref: 'adaguc-server'
+          format: 'table'
+          exit-code: '1'
+          ignore-unfixed: true
+          vuln-type: 'os,library'
+          trivyignores: .trivyignore
+#          severity: 'CRITICAL,HIGH'
```

Comment:
What does `severity` do, does it set a threshold and fails if exceeded? What is the default?
If `trivy` finds a vulnerability, does this fail the pipeline?

Reply:
I haven't look at the license stuff in detail. I think these messages are related to [license scanning](https://aquasecurity.github.io/trivy/v0.33/docs/licenses/scanning/), but not 100% sure. We don't have that turned on at the moment.